projects/sbb-esta/angular-maps/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@sbb-esta/angular-maps",
+  "version": "1.0.0",
```

Comment:
I recommend assigning `0.0.0-PLACEHOLDER` as version. This way it will have the same version as the other packages in a release.